precache/u1/app.js

```diff
@@ -0,0 +1,137 @@
+// Import libraries
+const express = require('express')
+
+// Setup app
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+// localhost:3000
+
+app.get('/', (req, res) => {
+  res.send('Better world')
+})
+
+// localhost:3000/html
+
+app.get('/html', (req, res) => {
+  res.send(`
+      <html><body>
+        <h1>Technologies</h1>
+        <ul>
+          <li>JavaScript</li>
+          <li>Node</li>
+          <li>Express</li>
+        </ul>
+      </body></html>
+    `)
+})
+// localhost:3000/json
+
+app.get('/json', (req, res) => {
+  res.json(
+    {
+      technologies: [
+        'JavaScript', 'Noodle', 'Express']
+    })
+})
+
+// localhost:3000/admin/grades
+app.get('/admin/grades', (req, res) => {
+  res.sendStatus(401)
+})
+
+// localhost:3000/api/grades
+
+app.get('/api/grades', (req, res) => {
+  res.status(401).send('Crepper! Oh man!')
+})
+
+// localhost:3000/boredom
+app.get('/boredom', (req, res) => {
+  res.status(404).send('Sorry-you just be a friend!')
+})
+// localhost3000/html-pic
+app.get('/html-pic', (req, res) => {
+  res.send(`
+    <html>
+      <body>
+    <h1>hello boii</h1>
+    <img src="/cat.jpg" width="300" />
+  </body>
+  </html>
+  `)
+})
+
+// Start server
+const PORT = 3000
+app.listen(PORT, () => console.log(
+  `listening on http://localhost:${PORT}`))
```

Comment:
`app.listen(...)` ควรอยู่ท้ายไฟล์ โปรดย้ายสิ่งนี้ไปยังจุดสิ้นสุด
`app.listen(...)` should be at the end of the file. Please move this to the end.

Reply:
คุณได้แก้ไขปัญหานี้แล้ว!
You have fixed this!